OpenOversight/app/main/views.py

```diff
@@ -140,9 +140,6 @@ def index():
 def set_session_timezone():
     if KEY_TIMEZONE not in session:
         session.permanent = True
-        current_app.permanent_session_lifetime = datetime.timedelta(
```

Comment:
Removing this since I think it changes the app settings, not just this particular session, potentially influencing other stuff and since the session is stored on the client side (as cookie) the default of 31 days seems to be fine

Reply:
Good to know! Appreciate that insight.